bugbug/models/bugtype.py

```diff
@@ -28,15 +30,52 @@
     "crashreportid": "crash",
     "perf": "performance",
 }
-KEYWORD_LIST = sorted(set(KEYWORD_DICT.values()))
+
+# For the moment, rca - XYZ is treated of bugtype XYZ,
+# so we don't need to store it in a dictionary.
+RCA_TYPES = [
+    "rca - external api failure",
+    "rca - logical error",
+    "rca - coding error",
+    "rca - unhandled exceptions",
+    "rca - testing error",
+    "rca - poor architecture",
+    "rca - logical error",
+    "rca - autoipcstream design error",
+    "rca: coding error",
+    "rca - undocumented change in os update",
+    "rca - design error",
+    "rca - ambiguous design",
+    "rca: coding error",
+    "rca - requirement",
+    "rca - unhandled exceptions",
+    "rca - logical error",
+    "rca - coding",
+    "rca - non-essential requirement",
+    "rca - coding error",
+    "rca - syntax error",
+    "rca - performance error",
+    "rca - testing error",
+    "rca - design error",
+    "rca - external software affecting firefox",
+    "rca - semantic error",
+    "rca - product decision",
+]
```

Comment:
Since there is no match between the bug type categories and the RCA categories, maybe better to define a separate model

Reply:
There is a match with "performance", but not more than that